etc/kayobe/kolla/config/prometheus/system.rules

```diff
@@ -78,4 +78,13 @@ groups:
       summary: "Host EDAC Uncorrectable Errors detected (instance {{ $labels.instance }})"
       description: "{{ $labels.instance }} has had {{ printf \"%.0f\" $value }} uncorrectable memory errors reported by EDAC in the last 5 minutes."
 
+  - alert: DiskSmartStatusUnhealthy
```

Comment:
This comes from Awesome alerts. Should we add a new file for our own alerts to avoid confusion?